core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -106,7 +111,8 @@ class AuthorizerIntegrationTest extends KafkaServerTestHarness {
     ApiKeys.LEAVE_GROUP.id -> ((resp: LeaveGroupResponse) => resp.errorCode()),
     ApiKeys.LEADER_AND_ISR.id -> ((resp: requests.LeaderAndIsrResponse) => resp.responses().asScala.find(_._1 == tp).get._2),
     ApiKeys.STOP_REPLICA.id -> ((resp: requests.StopReplicaResponse) => resp.responses().asScala.find(_._1 == tp).get._2),
-    ApiKeys.CONTROLLED_SHUTDOWN_KEY.id -> ((resp: requests.ControlledShutdownResponse) => resp.errorCode())
+    ApiKeys.CONTROLLED_SHUTDOWN_KEY.id -> ((resp: requests.ControlledShutdownResponse) => resp.errorCode()),
+	  ApiKeys.CREATE_TOPIC.id -> ((resp: requests.CreateTopicResponse) => resp.errors().asScala.find(_._1 == createTopic).get._2.code)
```

Comment:
@granthenke Example follows:

```scala

scala> Map("foo" -> 1, "bar" -> 3)
res7: scala.collection.immutable.Map[String,Int] = Map(foo -> 1, bar -> 3)

scala> res7.find(_._1 == "foo").get._2
res9: Int = 1

scala> res7.collectFirst { case (k, v) if k == "foo" => v }.get
res10: Int = 1
```

Also it would be better to use `getOrElse` instead of `get` to get a better error message in case the invariant is broken for whatever reason.